test/compute/vm.js

```diff
@@ -141,14 +141,19 @@ describe('VM', function() {
       name: DISK.formattedName
     });
 
+    var METADATA = {
+      disks: [
+        {
+          source: DEVICE_NAME,
+          deviceName: DEVICE_NAME
+        }
+      ]
+    };
+
     beforeEach(function() {
-      vm.metadata = {
-        disks: [
-          {
-            source: DEVICE_NAME,
-            deviceName: DEVICE_NAME
-          }
-        ]
+      vm.getMetadata = function(callback) {
+        vm.METADATA = METADATA;
```

Comment:
Can you kill the new line if it doesn't have any affect? I see future Stephen being confused.